curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6554d0332949b133a0b35eaa.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 6554d0332949b133a0b35eaa
+title: Step 63
+challengeType: 20
+dashedName: step-63
+---
+
+# --description--
+
+Call your function passing `text` and `custom_key` as the arguments. Store the return of the function call into a variable called `encryption`.
+
+# --hints--
+
+You should call `vigenere` passing `text` and `custom_key` as the arguments.
```

Comment:
I think it is better to go with `call <function_name>`, because the thing referencing the function is called, not the syntax defining it, or the result of the call.

That is, I would go with either:
a) "You should call `function_name` ...
b) "You should have `function_name()` ...